Xamarin.PropertyEditing.Mac/Controls/BindingEditor/BindingTypeSelectorControl.cs

```diff
@@ -0,0 +1,313 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using AppKit;
+using Foundation;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class BindingTypeSelectorControl : NSView
```

Comment:
Do we need to include this implementation? Doesn't appear to be used.

Reply:
Forgot to remove it. It was replaced by your TypeSelectorControl.